BrowserKit/Sources/Common/Theming/DefaultThemeManager.swift

```diff
@@ -36,11 +36,9 @@ public final class DefaultThemeManager: ThemeManager, Notifiable {
     private var sharedContainerIdentifier: String
 
     private var nightModeIsOn: Bool {
-        guard let isOn = userDefaults.object(forKey: ThemeKeys.NightMode.isOn) as? NSNumber,
-              isOn.boolValue == true
-        else { return false }
-
-        return true
+        let status = userDefaults.bool(forKey: ThemeKeys.NightMode.isOn)
+        print("RGB - \(status)")
```

Comment:
👀 
```suggestion
```

Reply:
I already removed it. Give me 1 min to review my own PR before you commets! You're toooooo quick! 😛  ❤️